Resources/Private/Translations/de/NodeTypes/LoginForm.xlf

```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='utf-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+	<file original="" product-name="Flowpack.Neos.FrontendLogin" source-language="en" datatype="plaintext">
```

Comment:
Don't know. I've created it with a tool (virtaal). Works in my projects as it is.